### ISSUE TITLE
Unable to find template plugin matching 'typescript'

### ISSUE THREAD

**Issue by danielleknudson** [dotansimha/graphql-code-generator#4913]
**Describe the bug**
Cannot generate types when the repo/directory name contains a "." (period) character.

**To Reproduce**
1. Clone example repo: https://github.com/danielleknudson/test2.0
2. `cd test2.0` && `yarn install`
3. Generate types: `yarn generate`

1. My GraphQL schema:

```graphql
  # This "Book" type defines the queryable fields for every book in our data source.
  type Book {
    title: String
    author: String
  }

  # The "Query" type is special: it lists all of the available queries that
  # clients can execute, along with the return type for each. In this
  # case, the "books" query returns an array of zero or more Books (defined above).
  type Query {
    books: [Book]
  }
```

2. My GraphQL operations:

```graphql
query GetBooks {
        books {
            title
            author
        }
}
```

3. My `codegen.yml` config file:

```yml
overwrite: true
schema: "server/index.js"
documents: "src/**/*.{ts,tsx}"
generates:
  src/generated/types.ts:
    plugins:
      - "typescript"
      - "typescript-operations"
      - "typescript-react-apollo"
```

**Expected behavior**
Types generated without issue

**Environment:**

- OS: Catalina 10.15.5
- `@graphql-codegen/...`: 1.17.10
- NodeJS: v12.18.0

**Additional context**
After digging into some of the cli code, I am certain the trouble generating types has to do with the directory name: `test2.0` in my case. My company has a repo with `2.0` in the directory name. After changing the name to `test2`, the types generate without errors. 

See `makeDefaultLoader()` in the cli/bin.js code.

**Comment by macgyver2k** [comment]
Yes, it's all about dots in the path. There is already a PR done [#4860](https://github.com/dotansimha/graphql-code-generator/pull/4860) by @adambrgmn. Let's hope the new release will be out soon.

**Comment by dotansimha** [comment]
@macgyver2k sorry for the delay. Will release today :)

**Comment by dotansimha** [comment]
Fixed in `@graphql-codegen/cli@1.18.0`
